### PR TITLE
Added basic tests for contacts

### PR DESCRIPTION
**Pull request by BranTurner**
Added basic tests for contacts:

1. Ensure user can see contacts
2. User can add contacts
3. User can add a note, activity, event, task, debt, and food preference to a contact
4. Test a contact can be deleted

There are many more contact tests I would like to add, and I feel the tests I'm pushing now can be refactored to make them more efficient. However, I don't have much more time to work on them today, and feel like some tests are better than no tests (especially when big changes are being merged!). 

I will hopefully improve this over the coming days, but I think this is a reasonable start.

### CODE REVIEW

File: tests/Feature/ContactTest.php [hunk 1]
```diff
@@ -0,0 +1,281 @@
...
+        $task['account_id'] = $user->account_id;
+
+        // Change keys to match database column names
+        $this->changeArrayKey('comment', 'description', $task);
+
+        $this->assertDatabaseHas('tasks', $task);
+    }
+
+    /*
+     * Waiting until encryption has been removed from gifts before this test
```
Comment (djaiss) [inline tests/Feature/ContactTest.php]: Encryption has been removed from the gifts table https://github.com/monicahq/monica/pull/249

**Comment by BranTurner** [comment]
@djaiss - updated :)

File: tests/FeatureTestCase.php [hunk 2]
```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ * Created by PhpStorm.
```
Comment (djaiss) [inline tests/FeatureTestCase.php]: Can you remove this comment section?

**Comment by BranTurner** [comment]
Apologies - PHPStorm add these automatically, and I keep forgetting to disable them!

**Pull request merged**